pxblue/react-native-component-library
# Update intellisense documentation comments

For PXBLUE-2251


#### Changes proposed in this Pull Request:
- Add intellisense comments where missing for
    - components
    - props
    - functions
- Update existing intellisense comments where needed to match more closely with docs
- Update some docs to be more consistent


#### Screenshots:
![image](https://user-images.githubusercontent.com/29152776/126371674-4dba24eb-7a52-4991-8294-8c575a2d144a.png)


#### To Test:
- clone the repository
- `yarn link:components`
- Open the App.tsx file and start typing out some components.
    - When you are getting hints for a component, you will see the detailed description
    - When typing out prop names, you will see detailed descriptions for the props
